src/test/java/com/solvd/carina/demo/SauceDemoMobileTest.java

```diff
@@ -0,0 +1,107 @@
+package com.solvd.carina.demo;
+
+import com.solvd.carina.demo.mobile.saucedemomobile.common.*;
+import com.zebrunner.carina.core.registrar.ownership.MethodOwner;
+import com.zebrunner.carina.utils.R;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class SauceDemoMobileTest extends AbstractSauceDemoScreenTest {
+
+    @Test
+    @MethodOwner(owner = "Andres")
+    public void loginTest() {
+        String username = R.TESTDATA.get("standard_username");
+        String password = R.TESTDATA.get("standard_password");
+
+        LoginScreenBase loginScreenBase = initPage(LoginScreenBase.class);
+        Assert.assertTrue(loginScreenBase.isOpened(), "Login screen is not opened");
+        loginScreenBase.typeUsername(username);
+        loginScreenBase.typePassword(password);
+        ProductsScreenBase productsScreenPage = loginScreenBase.clickLoginButton();
+        Assert.assertTrue(productsScreenPage.isOpened(), "Products screen is not opened");
+    }
+    @Test
+    @MethodOwner(owner = "Andres")
+    public void productAddedToTheCartTest() {
+        String productTitle = R.TESTDATA.get("second_product_name");
```

Comment:
Have you tried testing with your product_name = Sauce Labs Onesie? I think that product is further down the products screen

Reply:
Yes, To do it, it is necessary to swipe